packages/flame_3d/lib/src/resources/light/lighting_info.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flame_3d/resources.dart';
+
+class LightingInfo {
+  AmbientLight ambient = AmbientLight();
```

Comment:
```suggestion
  final ambient = AmbientLight();
```

I couldn't find it being reassigned anywhere

Reply:
it's reassigned here:
```
    device.lightingInfo.ambient = config.ambientLight;
```

Reply:
removed world config